Server/tools/cad/sketching.py

```
@@ -0,0 +1,156 @@
+    Beispiel: "XY", "YZ", "XZ"
+    """
+    try:
+        headers = get_headers()
+        endpoint = get_endpoints("cad")["draw_lines"]
+        data = {
+            "points": points,
+            "plane": plane
+        }
+        return send_request(endpoint, data, headers)
+    except requests.RequestException as e:
+        logging.error("Draw lines failed: %s", e)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)